docker/dev/backend/scripts/run-app

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+set -e
+set -u
+
+rm -rf "${OPENPROJECT_STORAGE_TMP__PATH:-tmp}/*"
+bundle exec rails server thin -b 0.0.0.0
```

Comment:
Why `thin` instead of `puma`?

Reply:
I took this from here https://github.com/opf/openproject/pull/8762/files#diff-ed0579942196f4cba4186b4e6e051eeeL16 and am too inexperienced with Ruby to have an opinion on which is better.